jinjalint/util.py

```diff
@@ -1,4 +1,5 @@
-import collections
+import collections.abc
+collections = collections.abc
```

Comment:
This should be `from collections.abc import Iterable` and then change line 9 to `if (isinstance(el, Iterable) and`